test/suites/upgrade/rollback-manual.robot

```diff
@@ -12,50 +12,56 @@ Test Tags           ostree
 
 
 *** Variables ***
-${USHIFT_HOST}              ${EMPTY}
-${USHIFT_USER}              ${EMPTY}
+${USHIFT_HOST}      ${EMPTY}
+${USHIFT_USER}      ${EMPTY}
 
-${FAKE_NEXT_MINOR_REF}      ${EMPTY}
+${UPGRADE_REF}      ${EMPTY}
 
 
 *** Test Cases ***
-Rollback
-    [Documentation]    Verifies that rolling back the system can be used to
-    ...    go back to older MicroShift version by restoring existing backup.
+Manual Rollback Does Not Restore Data
+    [Documentation]    Verifies that using rpm-ostree rollback will not
+    ...    restore data automatically: making sure admin needs to perform
+    ...    it manually.
+    ...
+    ...    Test expects that provided upgrade reference is within the same Y-stream,
+    ...    e.g. 4.14.0 -> 4.14.1, not 4.14.0 -> 4.15.0.
+    ...
+    ...    Test will upgrade the system with provided reference, create ConfigMap,
+    ...    roll system back to initial deployment, verify that ConfigMap exists,
+    ...    create additional ConfigMap, roll system back to upgrade deployment
+    ...    and verify that both ConfigMaps exist.
 
     ${initial_deploy_backup}=    Get Future Backup Name For Current Boot
     ${initial_deploy_id}=    Get Booted Deployment Id
 
-    Deploy Commit Not Expecting A Rollback    ${FAKE_NEXT_MINOR_REF}
+    Deploy Commit Not Expecting A Rollback    ${UPGRADE_REF}
     Backup Should Exist    ${initial_deploy_backup}
 
     ${upgrade_deploy_backup}=    Get Future Backup Name For Current Boot
     ${upgrade_deploy_id}=    Get Booted Deployment Id
 
-    # Creating a configmap while in "upgrade" deployment.
-    # It should not exist after first rollback (back to initial deployment),
-    # but should reappear after second rollback (back to "upgrade").
-    Run With Kubeconfig    oc create configmap -n default rollback-test
+    Oc Create    configmap -n default rollback-test
 
     Rollback System And Verify    ${upgrade_deploy_backup}    ${initial_deploy_id}
     ${rolled_back_deploy_backup}=    Get Future Backup Name For Current Boot
 
-    # Check that `oc get`'s error code is 1.
-    Run Keyword And Expect Error    1 != 0
-    ...    Oc Get    configmap    default    rollback-test
+    Oc Get    configmap    default    rollback-test
```

Comment:
The intent here isn't very clear. A keyword like "ConfigMap Should Exist" wrapping the oc get call would improve that.